multinet/api/utils/arango.py

```diff
@@ -2,16 +2,24 @@
 
 from functools import lru_cache
 from typing import Dict, List, Optional
+import uuid
 
 from arango import ArangoClient
 from arango.cursor import Cursor
 from arango.database import StandardDatabase
+from arango.http import DefaultHTTPClient
 from django.conf import settings
 
 
+class NoTimeoutHttpClient(DefaultHTTPClient):
+    """Extend the default arango http client, to remove timeouts for bulk data."""
+
+    REQUEST_TIMEOUT = None
```

Comment:
What could be the consequences of removing the timeout altogether? Could we end up with a zombie request thread somehow, or slowly leak memory in ArangoDB while Heroku terminates request threads otherwise, etc.?

If that could be a problem, what about a request timeout of something absurd, like 1 hour?

Reply:
I believe the main risk with not using a timeout is simply that theoretically the request could hang indefinitely. I agree, it makes sense to set _some_ timeout, even if it is very large. I think an hour is fine.

Reply:
See 5dee03b